src/Module/Admin/Item/Source.php

```diff
@@ -33,15 +33,7 @@ public static function content(array $parameters = [])
 	{
 		parent::content($parameters);
 
-		$a = DI::app();
-
-		$guid = null;
-		// @TODO: Replace with parameter from router
-		if (!empty($a->argv[3])) {
-			$guid = $a->argv[3];
-		}
-
-		$guid = $_REQUEST['guid'] ?? $guid;
+		$guid = basename($_REQUEST['guid']) ?? $parameters['guid'];
```

Comment:
When `$_REQUEST['guid']` is null then this will create a notice.

Reply:
Indeed.